default.nix

```diff
@@ -33,9 +34,10 @@ let
     pkgs = let
       rev = builtins.readFile (./nixpkgsForGhc + "/${ghcVersion}");
       sha256 = builtins.readFile (./generated + "/${buildName}/nixpkgsHashes/${rev}");
-    in import (fetchTarball {
-      url = "https://github.com/NixOS/nixpkgs/tarball/${rev}";
-      inherit sha256;
+    in import (fetchFromGitHub {
```

Comment:
Can do `pkgs.fetchFromGitHub` here directly, no need to add an argument to the file for that